CactbotOverlay/PluginLoader.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using RainbowMage.OverlayPlugin;
+using Advanced_Combat_Tracker;
+
+namespace Cactbot
+{
+    public class PluginLoader : IActPluginV1, IOverlayAddonV2
```

Comment:
I realized I forgot to explain this change. This class allows Cactbot to be added as a normal ACT plugin (it implements `IACTPluginV1`). The users just have to load it after `OverlayPlugin.dll` as it won't be able to find the overlay DLLs otherwise.
OverlayPlugin currently checks the list of loaded plugins as well as the existing `addons` folder to discover addons so both methods should work.

This way was easier for me (since I could just load `CactbotOverlay.dll` from VS' output directory) and it might make installation easier as the location of OverlayPlugin no longer matters and the installation works just like any other ACT plugin.